.env.example

```diff
@@ -53,3 +53,8 @@ VUE_APP_GENESYS_ID=
 VUE_APP_WEBCHAT_URL=
 VUE_APP_WEBCHAT_REASON=
 VUE_APP_WEBCHAT_STATUS_URL=
+
+#Document Record Service
+VUE_APP_DOC_API_KEY=
+VUE_APP_DOC_API_URL="https://test.api.connect.gov.bc.ca/doc-dev"
+VUE_APP_DOC_API_VERSION="/api/v1"
```

Comment:
You also need to add these to the vaults.json file.

Also, the API key will need to be added to the CI trigger config in GCP (for injecting the key into a preview build).